addons/html_editor/static/tests/html_field.test.js

```diff
@@ -1769,3 +1770,43 @@ describe("save image", () => {
         expect.verifySteps(["add_data: partner 1", "generate_access_token: 123"]);
     });
 });
+
+describe("translatable", () => {
+    test("should display translate button when html field is translatable", async () => {
+        expect.assertions(3);
```

Comment:
can remove this

Reply:
re-pushed. (1st push = requested changes, 2nd push = just rebase)